ui/app/templates/workspace/projects/project/app/logs.hbs

```diff
@@ -1,4 +1,4 @@
 {{page-title (concat @model.application.application "Logs")}}
 
 <h3>Application logs</h3>
-<LogStream @req={{@model}}></LogStream>
+<LogStream @req={{@model.request}}></LogStream>
```

Comment:
is this intentional? Doesn't seem related to the PR, and I see no other changes on the logs route

Reply:
It's to handle the breadcrumbs, I added the application name to the model so we could access it for that. If I remembered correctly, you can't use `modelFor` outside of the `model` function, otherwise I would have just grabbed it that way

Reply:
Ah gotcha, because the model for the route is now an object *containing* the request, where before it was just the request 👍

Reply:
With that clarification, I’m going to approve this.